src/tenderly/TenderlyService.ts

```diff
@@ -12,13 +12,16 @@ export const TENDERLY_DASHBOARD_BASE_URL = "https://dashboard.tenderly.co";
 
 export class TenderlyService {
   public static async verifyContracts(request: TenderlyContractUploadRequest) {
-    const tenderlyApi = TenderlyApiService.configureInstance();
+    let tenderlyApi = TenderlyApiService.configureAnonymousInstance();
+    let apiPath = "/api/v1/account/me/verify-contracts";
```

Comment:
Should this be `/me/` ? Would this work for organization principals? Maybe we need to pull this from the config like for pushing.